src/text-editor-component.js

```diff
@@ -819,7 +823,7 @@ class TextEditorComponent {
 
     const oldClassList = this.classList
     const newClassList = ['editor']
-    if (this.focused) newClassList.push('is-focused')
+    if (this.focused && this.isInputEnabled()) newClassList.push('is-focused')
     if (model.isMini()) newClassList.push('mini')
```

Comment:
Why this code assume `is-focused` only need to set only when `isInputEnabled()` was `true`?

Reply:
I need this unchanged, don't change meaning of `is-focused`.
Should allow `readonly` and `is-focused` buffer.
If some feature need exclude inputDisabled buffer from focused editor, do this by client side by checking state manually, don' bend core-meaning just because it's convenient.